### DISCUSSION TITLE
Can I cross-reference an issue from a different repository?

### DISCUSSION THREAD

**Discussion by lornat75** [robotology/community#3] [Q&A] [answered]
As per the title.

**Comment by lornat75** [comment, marked as answer]
Yes, you can include the name of the repository before the name, i.e.

This is dealt with in robotology/repository-name#issue_number

Thanks @pattacini

**Comment by traversaro** [comment]
Trick that saves some time: copy pasting directly the url of the issue result rendered in the same way. For example `https://github.com/robotology/codyco-superbuild/issues/43` becomes https://github.com/robotology/codyco-superbuild/issues/43 .

**Comment by rbonghi** [comment]
This is a good guide to understand principal feature about MarkDown
https://guides.github.com/features/mastering-markdown/

About **GitHub Flavored Markdown**
https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown
